.readthedocs.yml

```
@@ -0,0 +1,34 @@
+# .readthedocs.yml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the version of Python and other tools you might need
+build:
+  os: ubuntu-22.04
```

```suggestion
  os: ubuntu-lts-latest
```
This was updated in the template but I have yet to see it roll out to any package.